src/main/java/com/onepagecrm/models/serializers/CompanySerializer.java

```diff
@@ -96,11 +96,14 @@ public static Company fromJsonObject(JSONObject companyObject) {
         return companies;
     }
 
-    public static String toJsonObject(String contactId) {
+    public static String toJsonObject(String contactId, String linkedWithId) {
         JSONObject jsonObject = new JSONObject();
         if (contactId != null && !contactId.isEmpty()) {
             addJsonStringValue(contactId, jsonObject, CONTACT_ID_TAG);
         }
+        if (linkedWithId != null && !linkedWithId.isEmpty()) {
```

Comment:
@CillianMyles so if `linkedWithId` is null/empty, the key wouldn't be added at all?